README.md

```diff
@@ -1,64 +1,57 @@
+using [ReadTheDocs](https://readthedocs.org/).
+This site uses the [Sphinx Book](https://sphinx-book-theme.readthedocs.io) theme.
 This site is available here: https://researchcodereviewcommunity.github.io/dev-review/
 
-The following branches in this repository are important: 
+The following branches in this repository are important:
 - `main`: the markdown files corresponding to the current live version of the website
-- `gh-pages`: the static html site, automatically built by Hugo on new commits to `main` by [this script](.github/workflows/deploy.yml)
 
 ## Dependencies
-Install hugo extended dependency. 
-The "extended" hugo version is required to enable some features of [the LoveIt theme](https://hugoloveit.com/), such as mermaid diagrams.
-See commands to install hugo based on your Operating system.
-- macOS
+We use [`nox`](https://nox.thea.codes/en/stable/)
+as a task-runner to create development environments
+and to build the docs.
+There are many ways to install but we recommend the following:
+1. use a system package manager to install `pipx`
+2. use `pipx` to install `nox` so you can use it across projects
+3. finally ask `nox` to create a development environment
+
+- macOS/Linux
 ```
-brew install hugo
+brew install pipx
+pipx install nox
+nox -s dev
 ```
 - Windows
 ```
-choco install hugo -confirm
-```
-- GNU/Linux (Debian and Ubuntu)
+choco install pipx -confirm
+pipx install nox
+nox -s dev
 ```
-snap install hugo --channel=extended
-```
-
-See [documentation](https://gohugo.io/getting-started/installing/) for other options and furhter instructions. 
 
 ## Clone this repository
-After generating your SSH keys as suggested [here](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent). 
+After generating your SSH keys as suggested [here](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
 You can then clone the repository by typing (or copying) the following line in a terminal at your preferred path in your machine:
 ```
-git clone --recurse-submodules git@github.com:ResearchCodeReviewCommunity/dev-review.git
+git clone git@github.com:ResearchCodeReviewCommunity/dev-review.git
 ```
-If you've cloned this repo without the `--recurse-submodules` options, then you can initialise your local git submodule config with
 
-```
-git submodule init
-```
-and update it with
-```
-git submodule update
-```
+## Development
+To preview changes locally, you can build and serve the website locally.
 
-## Development 
-To preview changes locally, you can build and serve the website locally. 
-The `disableFastRender` option is required to render some features brought by the LoveIt theme, such as mermaid diagrams. 
 ```
-cd site/ && hugo serve --disableFastRender
+sphinx-build -nW --keep-going -b html doc/ doc/_build/html
+cd doc/_build/html; python -m http.server;cd ../../..
```

Comment:
I think this can be simplified to use a `nox` command (it is part of the reason to use `nox` in the first place from what I can tell):

```suggestion
nox -s docs -- serve
```

This worked on my machine at least.